src/Annotation/Tags.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Dealroadshow\K8S\Framework\Annotation;
```

Comment:
Annotation is name, used earlier.
PHP feature is called Attributes, so I think we should name this namespace `Attribute`